turbopack/crates/turbopack-ecmascript/src/chunk/item.rs

```diff
@@ -121,9 +121,9 @@ impl EcmascriptChunkItemContent {
             args.push("e: exports");
         }
         if self.options.stub_require {
-            args.push("z: require");
+            args.push("z: __turbopack_require_stub__");
         } else {
-            args.push("t: require");
+            args.push("t: __turbopack_require_real__");
```

Comment:
Do we need that at all? We could just call `require` directly.

Any indirect usage of `require` would cause these nested bundling problems again...